src/components/pages/BusinessCategoryPage/index.tsx

```
@@ -64,27 +64,15 @@ const ListItem = ({ type, item }: { type: string; item: any }) => {
       case 'sales':
         const itemSale = item as IPromotions
 
-        return (
-          <Link shallow href={`/sales/${itemSale.id}`} passHref>
-            <Sale item={itemSale} noHover />
-          </Link>
-        )
+        return <Sale item={itemSale} noHover />
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)